File: tl/src/main/java/com/github/badoualy/telegram/tl/api/upload/TLAbsFile.java
package com.github.badoualy.telegram.tl.api.upload;

import com.github.badoualy.telegram.tl.core.TLObject;

/**
 * Abstraction level for the following constructors:
 * <ul>
 * <li>{@link TLFile}: upload.file#96a18d5</li>
 * <li>{@link TLFileCdnRedirect}: upload.fileCdnRedirect#1508485a</li>
 * </ul>
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
public abstract class TLAbsFile extends TLObject {

    public TLAbsFile() {
    }
}

File: tl/src/main/kotlin/com/github/badoualy/telegram/tl/api/upload/TLAbsFile.kt
package com.github.badoualy.telegram.tl.api.upload

import com.github.badoualy.telegram.tl.core.TLObject

/**
 * Abstraction level for the following constructors:
 * * [upload.file#96a18d5][TLFile]
 * * [upload.fileCdnRedirect#ea52fe5a][TLFileCdnRedirect]
 *
 * @author Yannick Badoual yann.badoual@gmail.com
 * @see <a href="http://github.com/badoualy/kotlogram">http://github.com/badoualy/kotlogram</a>
 */
abstract class TLAbsFile : TLObject()
